nemo/collections/asr/models/ctc_models.py

```diff
@@ -185,7 +185,9 @@ def transcribe(self, paths2audio_files: List[str], batch_size: int = 4, logprobs
                         for idx in range(logits.shape[0]):
                             hypotheses.append(logits[idx][: logits_len[idx]])
                     else:
-                        hypotheses += self._wer.ctc_decoder_predictions_tensor(greedy_predictions)
+                        hypotheses += self._wer.ctc_decoder_predictions_tensor(
+                            greedy_predictions, predictions_len=logits_len
```

Comment:
@okuchaiev @vsl9 Please check if this is fine.

Reply:
Do all models return logits_len?

Reply:
Yes, all CTC_Models return it here : 
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/models/ctc_models.py#L390

It can be None and when it is None, it is ignored during WER calculation. But our current CTC models actually return the correct lengths.

For example, Jasper/QN return the lengths which are given by encoder here: 
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/modules/conv_asr.py#L192

Previously we did not use the lenghts return by the encoder at all in our codes. This would be the first time to actually use those.

Reply:
It seems fine then. Theoretically, the Jasper QuartzNet could make predictions past the current returned value due to the receptive field, but I don't think this makes a large practical difference